policy/policy-authority-data.go

```diff
@@ -169,9 +169,7 @@ func (padb *PolicyAuthorityDatabaseImpl) allowedByWhitelist(host string) bool {
 
 // CheckHostLists will query the database for white/blacklist rules that match host,
 // if both whitelist and blacklist rules are found the blacklist will always win
-// Returns errNotWhitelisted, errBlacklisted, or errDBFailure for the
```

Comment:
Wah sorry I'll rebase and fix tomorrow

On Tuesday, February 16, 2016, Jacob Hoffman-Andrews <
notifications@github.com> wrote:

> In policy/policy-authority-data.go
> <https://github.com/letsencrypt/boulder/pull/1482#discussion_r53101018>:
>
> > @@ -169,9 +169,7 @@ func (padb *PolicyAuthorityDatabaseImpl) allowedByWhitelist(host string) bool {
> >
> >  // CheckHostLists will query the database for white/blacklist rules that match host,
> >  // if both whitelist and blacklist rules are found the blacklist will always win
> > -// Returns errNotWhitelisted, errBlacklisted, or errDBFailure for the
>
> Sorry I missed this the last time around: Looks like you deleted this
> comment by accident.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/letsencrypt/boulder/pull/1482/files#r53101018>.
>


-- 


Jessie Frazelle
4096R / D4C4 DD60 0D66 F65A 8EFC  511E 18F3 685C 0022 BFF3
pgp.mit.edu <http://pgp.mit.edu/pks/lookup?op=get&search=0x18F3685C0022BFF3>